casper/src/main/scala/coop/rchain/casper/api/BlockAPI.scala

```diff
@@ -83,11 +83,12 @@ object BlockAPI {
     )
 
   def getListeningNameDataResponse[F[_]: Sync: MultiParentCasperRef: Log: SafetyOracle: BlockStore](
+      depth: Long,
```

Comment:
`depth` should be an `Int` if it is the number of blocks back that you are going as opposed to the height that you with to stop at.

Reply:
So we can't print out a depth of 3 million?

Reply:
Sure you can, `Int.MaxValue = 2,147,483,647`, which is over 2 billion. The point is that this API should not support more than that. If you need more than that then you should just copy the state directly (like literally copying the files used by casper to store blocks); it would be much faster that way.

Reply:
Oh lol. I think for the past 2 years I've been thinking 2.1 million is the max for ints. Whoops.